### PR TITLE
give roundstart borgs names

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
roundstart borgs get a name from the modified autoborg dataset
borgs made during the round are unaffected

## Why / Balance
funny + they were unused + borgs should have names that arent just "cyborg" roundstart that sucks

## Technical details
no

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- tweak: Roundstart borg jobs get funny names.

### CODE REVIEW

**Comment by LankLTE** [comment]
Honestly they should really use the character’s name but this works as an in-between solution

**Comment by deltanedas** [comment]
people that leave borg on low with a human char shouldnt use chars name, which is something i do

**Comment by LankLTE** [comment]
ok then don't do that, roundstart Borgs should have some way to get the name they want instead of being named "beepface IV" or "Cyborg"

**Comment by deltanedas** [comment]
come to think of it how does the game even replace urist mchands with the characters name...

**Comment by UbaserB** [comment]
We should do it like ss13 where theres a secondary naming option for whenever you become a borg

**Comment by deltanedas** [comment]
and mime and clown...

**Comment by EmoGarbage404** [comment]
i would rather a different dataset is used, as this one has quite a few joke names that could be kinda weird as defaults.

**Comment by mirrorcult** [comment]
yeah this dataset is for the malf AI's autoborger which has some funny names in it, i dont think this would be good as a roundstart name dataset

**Comment by deltanedas** [comment]
ok time to make malf :trollface:

**Comment by deltanedas** [comment]
@EmoGarbage404 using a new dataset with the malfier named removed

File: Resources/Prototypes/Datasets/Names/borg.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - Head of Robots
+  - Lvl 99 Firestarter
+  - EVA 1
+  - You but stronger
+  - Hat Simulator
+  - i died pls restart
+  - POTENT HAMS
+  - V.A.L.I.D.S.
+  - Remove Silicons
+  - say ";help in core"
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Datasets/Names/borg.yml:47]: this dataset is still full of shitposts. just make it normal names.

Comment (deltanedas) [reply Resources/Prototypes/Datasets/Names/borg.yml]: any more that need to go

**Comment by deltanedas** [comment]
merg

**Comment by mirrorcult** [comment]
the names in this are still pretty bad i think i'd rather just write a bunch from the ground up lol

**Comment by deltanedas** [comment]
too much effort when most of them are still funy

**Comment by Partmedia** [comment]
Closing because some of these names are pretty bad and the submitter doesn't want to improve them. Please re-open if you plan on improving these names.

**Comment by deltanedas** [comment]
then point out which ones are bad...

**Comment by Emisse** [comment]
yes i will compile a list on discord for you you just need to join the discord joinjoin the discord join the discord join the discord

**Comment by Partmedia** [comment]
I think the only appropriate round-start borg names from this list are:

  - Bob
  - Joe
  - Beep
  - Beep II
  - Tour Guide-otron
  - Boombox
  - Taffy
  - Boop
  - Boop II
  - Buzz
  - Buzz II
  - Toaster
  - Head of Robots
  - EVA 1

**Comment by deltanedas** [comment]
> yes i will compile a list on discord for you you just need to join the discord joinjoin the discord join the discord join the discord

no killer

**Comment by deltanedas** [comment]
@Partmedia reopen

File: Resources/Prototypes/Datasets/Names/borg.yml [hunk 2]
```diff
@@ -4,38 +4,15 @@
   values:
   - Bob
   - Joe
-  - Toiletron 9000
```
Comment (lzk228) [inline Resources/Prototypes/Datasets/Names/borg.yml]: i think this one is actually cool

Comment (deltanedas) [reply Resources/Prototypes/Datasets/Names/borg.yml]: illegal

**Comment by UbaserB** [comment]
Suggestions for borg names: (feel free to take any of these)
- Megabyte
- Scrap
- Fake-Drone
- Firewall
- Guybot
- Processing Device
- Waste Disposal
- Trashbin
- Microwave
- Annihilator
- Tinker
- Epsilon
- Omega
- Neon
- AI-bot

I am unoriginal so most of these are taken from random generators or lists online.

**Comment by forgotmyotheraccount** [comment]
Probably beyond the scope of this PR but I think it would be best if there two lists which are combined to create a randomized borg name. The first half consists of nouns and adjectives ("Boop", "Beep", "Toaster", etc), with the latter half consisting of "bot", "otron", "drone", etc. And then maybe slap a number on the end of it. So you'd end up with names like this:

- Beepbot 200
- Junkotron 500
- Robustinator 9000

Specialized borgs would have their own list of nouns and adjectives, so a roundstart janiborg might have names like:

- Trashbot 3000
- Cleaninator 8000
- Slipotron 5000

**Comment by lzk228** [comment]
Mr. Robist
Mini-Ripley
Robotron
C-3-D2
R2-PO
Robby
Robobot
Sonny
Autobot
Makeshifter
Beebee
Wheetley
Twin
Keller

**Comment by metalgearsloth** [comment]
Sure for now customisation later.

**Comment by CatMagic1** [comment]
Skeeboop
Mr. Roboto
Powernet
Beatbot
Deceptiborg
Buddy Bot
Empty Chassis
Le Borgue
Toaster 2.0
Borganic Lifeform
Stevie
Automaton
Beepers
Cute-Bot
Be-Nice the Borg

**Pull request merged**